PostPairsTrading.md

```diff
@@ -1,22 +1,15 @@
 # A Match Made in Trading: Step-by-Step Pairs Trading Guide
 
-KDB+/Q stands out as **a powerful tool in finance**, renowned for its ability to handle vast volumes of real-time data amidst the relentless dynamics of the market. In this article, we embark on an insightful exploration of Pairs Trading and its implementation in Q, offering a comprehensive guide to one of the most popular strategies in the trading world.
+KDB+/Q stands out as **a powerful tool in finance**, renowned for its ability to handle vast volumes of real-time data amidst the relentless dynamics of the market. In this article, we embark on an insightful exploration of _Pairs Trading_, one of the most popular strategies in the trading world, and its implementation in Q.
 
-Our objective is to provide **an implementation of a Pair Trading strategy in q**, which includes identifying cointegrated indexes, implementing a model to calculate spreads, making it work in real-time, and integrating it with the tick architecture.
+In order to achieve this exploration, we have outlined the following steps:
+* Identifying cointegrated indexes, i.e., related pairs
+* Implementing a model to calculate their spreads
+* Visualizing the approach in real-time
 
-We'll proceed methodically, ensuring each question leads to a comprehensive answer. To start, we'll contextualize our current situation by addressing key questions such as **"What do we know about the market and how can we benefit from it?"**. This will lay the foundation for constructing a real-time simulated environment on Pairs Trading that will exemplify everything we have explained thus far.
+We will use the [_Tick Architecture_](https://github.com/KxSystems/kdb-tick), the typical setup found in kdb+ systems, to introduce, contextualize and guide each step. As we acknowledge that the typical reader is a quant interested in learning the virtues of q, a concise introduction to this architecture will be provided. We have also aimed to briefly introduce pairs trading so that other developers can follow and benefit from this text. Please, feel free to skip the following section if you are familiar with it.

```

Comment:
Is it KDB, kdb or Kdb? On the same note, is it Q or q? Very minor but this should be consistent

Reply:
I always use _kdb+_ and _q_, I think this is the most standard way of referring to them, taking into account Whitney's convention, but you're right, I'll adapt to the original style from the text: _KDB+_ and _Q_.